src/uucore/src/lib/macros.rs

```diff
@@ -21,6 +21,14 @@ macro_rules! executable(
     })
 );
 
+#[macro_export]
+macro_rules! show(
+    ($err:expr) => ({
+        uucore::error::set_exit_code($err.code());
+        eprintln!("{}: {}", executable!(), $err);
```

Comment:
I think the use of `executable!()` means that this macro can only be used in individual utilities, but not in `uucore`, because `executable!()` looks at the path of the current file and guesses the utility name from that.
Maybe we can instead change how `executable!()` works and make it return the actual command that was executed?

Reply:
That's true. In what kind of situations would you like to use it in `uucore`? Making a more robust implementation won't be easy, because it needs to work both for the individual binaries and the `coreutils` binary.

Reply:
I didn't have a particular use case in mind, so I guess we can leave it as it is.